src/structures/cdn.ts

```diff
@@ -1,11 +1,13 @@
 import { ImageFormats, ImageSize } from '../types/cdn.ts'
 
+/** Function to get Image URL from a resource on Discord CDN */
 export const ImageURL = (
   url: string,
-  format: ImageFormats,
-  size?: ImageSize | 128
+  format: ImageFormats | undefined = 'webp',
```

Comment:
If `format` is undefined, it will become `'webp'` because you set its default value.